src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/appsettings.json

```diff
@@ -85,5 +85,17 @@
         ]
       }
     }
+  },
+  "ProvinceList": {
```

Comment:
@ytqsl Are you ok with loading provinces in configuration? Should we load them from the dynamics side?

Reply:
@qsl-deepak I've discussed this with Yossi, and currently, there's no decision to create a separate entity for provinces in Dynamics. We should maintain this as a string in the backend and use an enum in the frontend instead. Therefore, this subtask would be unnecessary since it pertains solely to frontend work.

Reply:
@farzadnadiri - Thanks for the update. closing this PR.